gen-app-builder/chat-app/site/.firebaserc

```diff
@@ -0,0 +1,15 @@
+{
+  "projects": {
+    "default": "koverholt-devrel-355716"
```

Comment:
Remove References to your own project, maybe replace with `YOUR_PROJECT_ID`

Also add steps to readme to replace those values in specific filenames

Reply:
Thanks for catching this. I generalized the values in 356e5cf.

Normally I don't commit .rc files and such, but committing it here captures a potentially confusing step when deploying to firebase. Nonetheless, I will give this a couple of more passes when writing up the docs in the README and make this as easy/clean as possible.